.gitignore

```diff
@@ -1,10 +1,338 @@
+[Dd]ebug/
+[Rr]elease/
+x64/
+build/
+[Bb]in/
+[Oo]bj/
+
+# MSTest test Results
+[Tt]est[Rr]esult*/
+[Bb]uild[Ll]og.*
+*_i.c
+*_p.c
+*.ilk
+*.meta
+*.obj
+*.pch
+*.pdb
+*.pgc
+*.pgd
+*.rsp
+*.sbr
+*.tlb
+*.tli
+*.tlh
+*.tmp
+*.tmp_proj
+*.log
+*.vspscc
+*.vssscc
+.builds
+*.pidb
+*.log
+*.scc
+
+# Visual C++ cache files
+ipch/
+*.aps
+*.ncb
+*.opensdf
+*.sdf
+*.cachefile
+
+# Visual Studio profiler
+*.psess
+*.vsp
+*.vspx
+
+# Guidance Automation Toolkit
+*.gpState
+
+# ReSharper is a .NET coding add-in
+_ReSharper*/
+*.[Rr]e[Ss]harper
+
+# TeamCity is a build add-in
+_TeamCity*
+
+# DotCover is a Code Coverage Tool
+*.dotCover
+
+# NCrunch
+*.ncrunch*
+.*crunch*.local.xml
+
+# Installshield output folder
+[Ee]xpress/
+
+# DocProject is a documentation generator add-in
+DocProject/buildhelp/
+DocProject/Help/*.HxT
+DocProject/Help/*.HxC
+DocProject/Help/*.hhc
+DocProject/Help/*.hhk
+DocProject/Help/*.hhp
+DocProject/Help/Html2
+DocProject/Help/html
+
+# Click-Once directory
+publish/
+
+# Publish Web Output
+*.Publish.xml
+*.pubxml
+
+# NuGet Packages Directory
+## TODO: If you have NuGet Package Restore enabled, uncomment the next line
+#packages/
+# Windows Azure Build Output
+csx
+*.build.csdef
+
+# Windows Store app package directory
+AppPackages/
+
+# Others
+sql/
+*.Cache
+ClientBin/
+[Ss]tyle[Cc]op.*
+~$*
+*~
+*.dbmdl
+*.[Pp]ublish.xml
+*.pfx
+*.publishsettings
+
+# RIA/Silverlight projects
+Generated_Code/
+
+# Backup & report files from converting an old project file to a newer
+# Visual Studio version. Backup files are not needed, because we have git ;-)
+_UpgradeReport_Files/
+Backup*/
+UpgradeLog*.XML
+UpgradeLog*.htm
+
+# SQL Server files
+App_Data/*.mdf
+App_Data/*.ldf
+
+## Espresso Web Editor
+*.esproj
+*.espressostorage
+
+## Sublime Text
+*.sublime-workspace
+*.sublime-project
+
+#sftp/ftp plugin scary things!
+sftp-config.json
+ftpsync.settings
+
+## Atom.io
+deployment-config.json
+.remote-sync.json
+
+## Brackets Editor
+.brackets.json
+
+## Visual Studio Code
+.settings/
+.vscode/
+
+## Netbeans
+.netbeans.xml
+
+### Languages & Tools
+
+## Python
+*.py[co]
+
+# Packages
+*.egg
+*.egg-info
+dist/
+build/
+eggs/
+parts/
+var/
+sdist/
+develop-eggs/
+.installed.cfg
+
+# Installer logs
+pip-log.txt
+
+# Unit test / coverage reports
+.coverage
+.tox
+
+#Translations
+*.mo
+
+#Mr Developer
+.mr.developer.cfg
+
+## SSL
+*.key
+
+## NodeJS
+npm-debug-log
+
+## Gulp/Grunt
+node_modules
+
+## cTags
+.tags
+.tags_sorted_by_file
+
+## Bower
+bower_components
+
+## Browser-Sync
+bs-config.js
+
+### CMSs & Frameworks
+
+## Drupal
+# Ignore configuration files that may contain sensitive information.
+sites/*/settings*.php
+
+# Ignore paths that contain user-generated content.
+sites/*/files
+sites/*/private
+
+## Wordpress
+wp-config.php
+wp-content/uploads
+wp-content/upgrade
+
+# Grav Caches
+cache/compiled/
+cache/doctrine/
+cache/gpm/
+cache/twig/
+
+# Others
+.idea
+node_modules/
+temp/
 reports/
-tmp/
-test-results/
+videos/
```

Comment:
The gitignore has a lot of stuff from tools and programming languages that are not used, it's too much overkill 😅 

Let's add just the necessary lines plus the ones you added regarding `Logs`, `.env` and `Runtime data`

Reply:
yep!
`.gitignore` with just what we need.